bpmn-model/src/test/java/io/zeebe/model/bpmn/validation/ZeebeValidationTest.java

```diff
@@ -112,7 +115,9 @@
       {
         Bpmn.createExecutableProcess().startEvent().signal("signal").endEvent().done(),
         Arrays.asList(
-            expect(StartEvent.class, "Start event must be one of the following types: none, timer"),
+            expect(
```

Comment:
As we're starting to have more start event tests, I would extract them to a specific `ZeebeStartEventValidationTest` file, like we did for the other elements (e.g. `ZeebeTimerValidationTest`)